docs/pipeline/mag/eva.md

```diff
@@ -0,0 +1,15 @@
+# nf-core/configs: eva mag specific configuration
+
+Extra specific configuration for mag pipeline
+
+## Usage
+
+To use, run the pipeline with `-profile eva`.
+
+This will download and launch the eager specific [`eva.config`](../../../conf/pipeline/mag/eva.config) which has been pre-configured with a setup suitable for the MPI-EVA cluster.
```

Comment:
```suggestion
This will download and launch the mag specific [`eva.config`](../../../conf/pipeline/mag/eva.config) which has been pre-configured with a setup suitable for the MPI-EVA cluster.
```